lib/authenticate.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+'use strict'
+
+/* ::
+import type {JWTPayload} from '../types.js'
+*/
+
+// Use boom for error handling
+const Boom = require('boom')
+// Use jsonwebtoken for verifying access token sent in header of request
+const jwt = require('jsonwebtoken')
+
+const ensureEnvVariable = require('./ensure-env-var.js')
+
+function authenticate (
+  authorization /* : string */
+) /* : Promise<JWTPayload> */ {
+  const audience = ensureEnvVariable('JWT_AUDIENCE')
+  const issuer = ensureEnvVariable('JWT_ISSUER')
+  const jwtSecret = ensureEnvVariable('JWT_SECRET')
+  // authorization should be 'Bearer thisismyaccesstoken'
+  const auth = authorization.split(' ')
+  const token = auth[1]
```

Comment:
Is it worth checking that `token` is a string and not `undefined` before proceeding? e.g. when there's no Authorization header or it's empty or not in the expected format?